build-pipeline/dotnet-docker-linux-images.json

```diff
@@ -125,7 +125,7 @@
       "value": "microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20170710090943"
     },
     "image-builder.args": {
-      "value": "--command build --manifest manifest.json --path $(PB.image-builder.path) --test-var Filter=$(PB.image-builder.path) --push --username $(PB.docker.username) --password $(PB.docker.password)",
+      "value": "--command build --manifest manifest.json --repo microsoft/dotnet-nightly --path $(PB.image-builder.path) --test-var Filter=$(PB.image-builder.path) --push --username $(PB.docker.username) --password $(PB.docker.password)",
```

Comment:
The intent here is that only `microsoft/dotnet-nightly` will be built in the official build, and `microsoft/nanoserver-insider-dotnet` must be built manually, right?